mllib/src/test/scala/org/apache/spark/ml/classification/DecisionTreeClassifierSuite.scala

```diff
@@ -262,6 +262,9 @@ class DecisionTreeClassifierSuite
       assert(Vectors.dense(rawPred.toArray.map(_ / sum)) === probPred,
         "probability prediction mismatch")
     }
+
+    ProbabilisticClassifierSuite.probabilisticClassifierGenericTest[
```

Comment:
We should use a more descriptive name for this test. How about `ProbabilisticClassifierSuite.testPredictMethods`? @jkbradley may have other suggestions too.